### PR TITLE
Feature/redux component

### PR DESCRIPTION
**Pull request by emarkees**
### For this Milestone I:

✅ Used GitFlow
✅ Used linters
✅ Set up a React App
✅ Used React components
✅ Used React props
✅ Used React state
✅ Handled events in a React app
✅ Used React Router
✅ Understood the concept of state management
✅ Used store, actions, and reducers in React
✅ Setup a Redux store
✅ Created a slice to save state and reducers
✅ Connected React and Redux

Kindly review this milestone and give me a star ⭐ as a mark of encouragement.

Thanks

### CODE REVIEW

File: src/components/BookItem.jsx [hunk 1]
```diff
@@ -48,14 +56,15 @@ const BookItem = ({ bookProp }) => {
           <button type="button">Comment</button>
           {' '}
           |
-          <button type="button">Remove</button>
+          <button type="button" onClick={handleRemove}>Remove</button>
           |
-          <button type="button" onClick={handleEdit}> Edit </button>
+          <button type="button" onClick={handleEdit}>Edit</button>
```
Comment (Kamba56) [inline src/components/BookItem.jsx:61]: - [ ] The remove button is not visible in your app hence I can't test if the remove function is working. Kindly fix this, please.
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @emarkees, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job using redux to store items!👏👏👏
- [x] Great job using useSelector to access store

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookItem.jsx (hunk 1 above)
Comment (GzimAsani) [reply src/components/BookItem.jsx:61]: -  The button is now visible and I am able to delete books

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**